src/main/java/seedu/sellsavvy/logic/Messages.java

```diff
@@ -19,6 +19,7 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
```

Comment:
```suggestion
                "Multiple values specified for the following single-valued field(s): ";
                
```

Suggested whitespace for tidiness.